tests/Tests/Graph/DOT.elm

```diff
@@ -46,7 +46,7 @@ all =
   0 [label="Welcome"]
   1 [label="To"]
   2 [label="Web"]
-  3 [label="\\"GraphViz\\"!"]
+  3 [label=""GraphViz"!"]
```

Comment:
This is really a subtle breaking change I'm not willing to make. Try to properly fix https://github.com/elm-community/graph/pull/21/files#diff-4f0eb98e08fb690b622716455626060aL128.

Reply:
I don't think that actually changes the behavior.  The change on L128 just wraps the string in quotes, which is what `Basics.toString` was doing since it was already a String value.

I think it is more of an issue with how escaping works in the test file.  I changed it to `"\\\"GraphViz\\\"!"` which will turn into the string `\"GraphViz\"!`, which is what I think the desired output is. I have tested this in https://dreampuf.github.io/GraphvizOnline/ and it works as I would expect.  

Let me know if I'm missing something.